public/js/app.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+  $('#preview').on('click', function() {
+    $('#sudokustring').val().split("").forEach(function(number, i) {
+      $($('td input')[i]).val(number);
+    });
+  });
```

Comment:
I would consider wrapping some of this functionality into a module, I consider this page to be like a controller so we want to keep logic in here to an absolute minimum